src/preact/base-element.js

```diff
@@ -725,6 +727,21 @@ function collectProps(Ctor, element, ref, defaultProps) {
       devAssert(def.attrs);
       value = def.parseAttrs(element);
     }
+    if (def.attrPrefix) {
+      const currObj = {};
+      const attrs = element.attributes;
+      for (let i = 0; i < attrs.length; i++) {
+        const attrib = attrs[i];
+        if (attrib.name.startsWith(def.attrPrefix)) {
```

Comment:
Please use `startsWith` from `string.js`.

Reply:
It seems to have been removed from `master` in a previous commit unfortunately, which is why I switched to the JS version: https://github.com/ampproject/amphtml/commit/246b09c4f8c80a621289d3b9dcb67436af94a291#diff-82a006d4e204ffc6dc16d4edac4f8cf5d186da3299498e5bc8bcc97ae8944a37